README.md

```diff
@@ -9,22 +9,13 @@
 [![GolangCI](https://golangci.com/badges/github.com/cosmos/gaia.svg)](https://golangci.com/r/github.com/cosmos/gaia)
 [![riot.im](https://img.shields.io/badge/riot.im-JOIN%20CHAT-green.svg)](https://riot.im/app/#/room/#cosmos-sdk:matrix.org)
 
-This repository hosts `Gaia`, the first implementation of the Cosmos Hub.
+This repository hosts `Gaia`, the first implementation of the Cosmos Hub based on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
 
 **Note**: Requires [Go 1.13+](https://golang.org/dl/)
 
-**DISCLAIMER**: The current version of Gaia running the Cosmos Hub (v0.34.x) is
```

Comment:
Why was this removed? I added it because we kept getting github issues created stating their binary wasn't working and that was because they were using this repo!

Reply:
Right @alexanderbez, I thought I opened it as a draft, my bad. It should not be merged before the hub upgrade

Reply:
got it